Frontend/src/App.jsx

```diff
@@ -58,92 +57,118 @@ function App() {
 
     getWords()
 
-  }, [(fireGetWords)])
-
+  }, [fireGetWords])
```

Comment:
The dependencies of `useEffect` is an array, so it can have multiple values that the effect depends on. This means that this could be:

```ts
}, [length, unique]);
```

However, it's generally recommended to not use `useEffect()` unnecessarily. In this case, you could initiate the request (by calling the `getWords()` function (if it was defined somewhere else) at the point when the length or unique is changed.